CHANGELOG.md

```diff
@@ -1,8 +1,7 @@
 # Change Log
 
 ## [0.7.2](https://github.com/maxpoint/conda-mirror/tree/0.7.2) (2017-05-23)
-
```

Comment:
The automated changelog generator removed it, so 🤷‍♂️ 

> Not sure how you like to do formatting here. 

I like to do none of the formatting and let the generator handle it 😀